hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/DBStoreBuilder.java

```diff
@@ -182,6 +183,17 @@ private void applyDBDefinition(DBDefinition definition) {
     }
   }
 
+  public DBStoreBuilder setMaxNumberOfOpenFiles(Integer maxNumberOfOpenFiles) {
```

Comment:
nit: Can you please move it to next to other setters?

Reply:
done